src/Catalyst.Core.Modules.Web3/ApiModule.cs

```diff
@@ -88,8 +89,10 @@ async void BuildCallback(IContainer container)
                                    .UseUrls(_apiBindingAddress)
                                    .UseWebRoot(webDirectory.FullName)
                                    .UseSerilog();
-                            }).Build();
-                    await host.StartAsync();
+                            }).RunConsoleAsync();
+
+                    //SIGINT is caught from kestrel because we are using RunConsoleAsync in HostBuilder, the SIGINT will not be received in the main console so we need to exit the process manually, to prevent needing to use two SIGINT's
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 236 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4942625)